apps/products-feed/src/modules/app-configuration/app-config.ts

```diff
@@ -1,34 +1,91 @@
 import { z } from "zod";
-import { UrlConfiguration } from "./url-configuration";
 
-export const s3BucketConfigurationSchema = z.object({
-  bucketName: z.string(),
-  secretAccessKey: z.string(),
-  accessKeyId: z.string(),
-  region: z.string(),
+const S3Config = z.object({
+  bucketName: z.string().min(1),
+  secretAccessKey: z.string().min(1),
+  accessKeyId: z.string().min(1),
+  region: z.string().min(1),
 });
 
-export type S3BucketConfiguration = z.infer<typeof s3BucketConfigurationSchema>;
+const UrlConfiguration = z.object({
```

Comment:
Do we want to write uppercase schema names?

Reply:
I like to follow pattern:
- schemas: `urlConfigurationSchema` - I'm adding echema because probably i will have conf data variable in the scope named urlConfiguration
- types: `UrlConfiguration` - drop schema from name, use upper case

Reply:
Okay will change